README.md

```diff
@@ -126,6 +126,49 @@ Try the following examples to learn more about Dapr's Java SDK:
 
 Please, refer to our [Javadoc](https://dapr.github.io/java-sdk/) website.
 
+#### Reactor API
+
+The Java SDK for Dapr is built using [Project Reactor](https://projectreactor.io/). It provides an asynchronous API for Java. When consuming a result is consumed synchronously, as in the examples above, the `block()` method is used.
```

Comment:
> as in the examples above
Sounds like this should say "below".

Reply:
The idea is above because the examples are referenced above.

Reply:
I reworded a little bit to make it clear.